docs/conf.py.in

```diff
@@ -16,7 +16,7 @@ templates_path = ['templates']
 source_suffix = '.rst'
 master_doc = 'index'
 project = u'OpenColorIO'
-copyright = u'2010, Sony Pictures Imageworks'
+copyright = u'2020, Academy Software Foundation'
```

Comment:
I think the entire copyright statement should actually be changed to `Copyright Contributors to the OpenColorIO Project.` (with no year). That's what we use in sources. @jmertic can you confirm if this makes sense for the website, or should it be ASWF in that case?

Reply:
Yes, the website copyright should align with the same one aligned to for the codebase.

You might also want to consider a footer similar to OpenVDB - check their website https://openvdb.org for that language, which also includes the CC BY 40 license statement